docs/source/guides/setting_config_options.ipynb

```diff
@@ -98,7 +98,7 @@
     "\n",
     "### Natural Language Threshold\n",
     "\n",
-    "The `natural_language_threshold` config variable helps control the distinction between `Categorical` and `NaturalLanguage` logical types during type inference. More specifically, this threshold represents the average string length that is used to distinguish between these two types. If the average string length in a column is greater than this threshold, the column is inferred as a `NaturalLanguage` column; otherwise, it is inferred as a `Categorical` column. The `natural_language_threshold` config variable defaults to `10`.\n",
+    "The `categorical_threshold` config variable helps control the distinction between `Categorical` and `Unknown` logical types during type inference. More specifically, this threshold represents the average string length that is used to distinguish between these two types. If the average string length in a column is greater than this threshold, the column is inferred as a `Unknown` column; otherwise, it is inferred as a `Categorical` column. The `categorical_threshold` config variable defaults to `10`.\n",
```

Comment:
The header above this section still says `Natural Language Threshold` and should be updated to `Categorical Threshold`